student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name, parent_permission)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] default values for `name` and `parent_permission` should be assigned here same way you did it in Person class, otherwise you will be getting argument error.